failed: TickSourceSpec

https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/6329/consoleText

```
[info] - must be possible to cancel *** FAILED *** (1 second, 853 milliseconds)
[info]   java.lang.AssertionError: assertion failed: received unexpected message OnNext(tick)
[info]   at scala.Predef$.assert(Predef.scala:170)
[info]   at akka.testkit.TestKitBase$class.expectNoMsg_internal(TestKit.scala:577)
[info]   at akka.testkit.TestKitBase$class.expectNoMsg(TestKit.scala:573)
[info]   at akka.testkit.TestKit.expectNoMsg(TestKit.scala:734)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectNoMsg(StreamTestKit.scala:516)
[info]   at akka.stream.scaladsl.TickSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$5$$anonfun$apply$mcV$sp$11.apply(TickSourceSpec.scala:94)
[info]   at akka.stream.scaladsl.TickSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$5$$anonfun$apply$mcV$sp$11.apply(TickSourceSpec.scala:86)
[info]   at akka.stream.testkit.Utils$.assertAllStagesStopped(Utils.scala:25)
[info]   at akka.stream.scaladsl.TickSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$5.apply$mcV$sp(TickSourceSpec.scala:86)
[info]   at akka.stream.scaladsl.TickSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$5.apply(TickSourceSpec.scala:86)
[info]   at akka.stream.scaladsl.TickSourceSpec$$anonfun$1$$anonfun$apply$mcV$sp$5.apply(TickSourceSpec.scala:86)
```

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/TickSourceSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/TickSourceSpec.scala
@@ -15,12 +15,9 @@ class TickSourceSpec extends StreamSpec {
   "A Flow based on tick publisher" must {
     "produce ticks" in assertAllStagesStopped {
       val c = TestSubscriber.manualProbe[String]()
-      Source.tick(1.second, 500.millis, "tick").to(Sink.fromSubscriber(c)).run()
+      Source.tick(1.second, 1.second, "tick").to(Sink.fromSubscriber(c)).run()
       val sub = c.expectSubscription()
-      sub.request(3)
-      c.expectNoMsg(600.millis)
-      c.expectNext("tick")
-      c.expectNoMsg(200.millis)
+      sub.request(2)
       c.expectNext("tick")
       c.expectNoMsg(200.millis)
       c.expectNext("tick")
@@ -84,13 +81,11 @@ class TickSourceSpec extends StreamSpec {
 
     "be possible to cancel" in assertAllStagesStopped {
       val c = TestSubscriber.manualProbe[String]()
-      val tickSource = Source.tick(1.second, 500.millis, "tick")
+      val tickSource = Source.tick(1.second, 1.second, "tick")
       val cancellable = tickSource.to(Sink.fromSubscriber(c)).run()
       val sub = c.expectSubscription()
-      sub.request(3)
+      sub.request(2)
       c.expectNoMsg(600.millis)
-      c.expectNext("tick")
-      c.expectNoMsg(200.millis)
       c.expectNext("tick")
       c.expectNoMsg(200.millis)
       c.expectNext("tick")

